src/components/Resource/Wizard/ModelWizard.tsx

```diff
@@ -92,14 +100,16 @@ const CreateSteps = ({
       {/* Step 1: Model Info */}
       <Stepper.Step label={editing ? 'Edit model' : 'Create your model'}>
         <Stack pos="relative">
-          <LoadingOverlay visible={isInitialLoading} />
+          <LoadingOverlay visible={isInitialLoading || isBountyFieldsInitialLoading} />
           <Title order={3}>{editing ? 'Edit model' : 'Create your model'}</Title>
           <ModelUpsertForm
-            model={model ?? templateFields}
+            model={model ?? templateFields ?? bountyFields}
             onSubmit={({ id }) => {
               if (editing) return goNext();
               router.replace(
-                `/models/${id}/wizard?step=2${templateId ? `&templateId=${templateId}` : ''}`
+                `/models/${id}/wizard?step=2${templateId ? `&templateId=${templateId}` : ''}${
+                  bountyId ? `&bountyId=${bountyId}` : ''
+                }`
```

Comment:
I see that we have a `getWizardUrl` method in this same file, maybe we can use that here
```suggestion
               getWizardUrl({ id, step: 2, templateId, bountyId })
```